crypto/dilithium/pqcrystals_dilithium_ref_common/poly.c

```diff
@@ -462,10 +405,13 @@ void poly_uniform_eta(ml_dsa_params *params,
     stream256_squeezeblocks(buf, 1, &state);
     ctr += rej_eta(params, a->coeffs + ctr, N - ctr, buf, STREAM256_BLOCKBYTES);
   }
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values. */
+  OPENSSL_cleanse(buf, sizeof(buf));
```

Comment:
missing ctr

Reply:
added in [c33b520](https://github.com/aws/aws-lc/pull/2017/commits/c33b520969414e82fa7fca5628f5c7af8f79195b)